site/_posts/2017-11-27-bugs-in-fix-out.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Last Week in ManageIQ: Bugs In Fix Out"
+author: h-kataria
+date: 2017-11-28
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+Hello there! This week's LWIMIQ is contributed by [Harpreet](https://github.com/h-kataria)
+Hope everyone had a wonderful time enjoying Thanksgiving feast with family and friends, and ofcourse shopping Black Friday deals. With Christmas and New Year right around the corner everyone must be in holiday mode, while our team at ManageIQ is working hard to get Gaprindashvili as stable as possible. Since we are in a bug fixing phase not much to report features/enhancement wise from my side.
```

Comment:
Since Thanksgiving (last week) is a US thing,..perhaps say, "hope everyone in the US had,..."
and
"ofcourse" to: "of course"